src/ontology/mondo-edit.obo

```diff
@@ -309579,17 +309579,21 @@ xref: Orphanet:1775 {source="GARD:0010905", source="MONDO:equivalentTo", source=
 xref: SCTID:74911008 {source="MONDO:equivalentTo", source="DOID:2729"}
 xref: UMLS:C0265965 {source="NCIT:C111802", source="ORDO:1775/e", source="MONDO:equivalentTo", source="Orphanet:1775", source="DOID:2729"}
 xref: UMLS:C1148551 {source="GARD:0002007", source="MONDO:notFoundInDiseaseSubset"}
-is_a: MONDO:0001713 {source="Orphanet:1775"} ! inherited aplastic anemia
-is_a: MONDO:0002254 {source="MONDO:Redundant", source="MONDO:cjm", source="NCIT:C111802/inferred"} ! syndromic disease
 is_a: MONDO:0015356 {source="Orphanet:1775"} ! hereditary neoplastic syndrome
-is_a: MONDO:0016382 {source="Orphanet:1775"} ! hereditary poikiloderma
-is_a: MONDO:0018035 {source="Orphanet:1775"} ! syndrome with combined immunodeficiency
 is_a: MONDO:0019287 {source="Orphanet:1775"} ! ectodermal dysplasia syndrome
-is_a: MONDO:0019289 {source="Orphanet:1775"} ! hyperpigmentation of the skin
-is_a: MONDO:0020063 {source="Orphanet:1775"} ! malformation syndrome with hamartosis
-is_a: MONDO:0020195 {source="Orphanet:1775"} ! excretory apparatus of the lacrimal system anomaly
-is_a: MONDO:0020204 {source="Orphanet:1775"} ! conjunctival tumor
+relationship: disease_has_feature HP:0000953 {source="Orphanet:1775"} ! Hyperpigmentation of the skin
+relationship: disease_has_feature HP:0001903 {source="Orphanet:1775"} ! Anemia
+relationship: disease_has_feature HP:0002721 {source="Orphanet:1775"} ! Immunodeficiency
+relationship: disease_has_feature HP:0010566 {source="Orphanet:1775"}
+relationship: excluded_subClassOf MONDO:0001713 {source="Orphanet:1775"} ! inherited aplastic anemia
+relationship: excluded_subClassOf MONDO:0002254 {source="MONDO:Redundant", source="MONDO:cjm", source="NCIT:C111802/inferred"} ! syndromic disease
```

Comment:
No need to exclude if you are just getting rid of redundancy and providing a more refined class. The purpose of excluded_subClassOf is to record we intentionally disagreed.